llvm/test/MachineVerifier/test_g_multiple_errors.mir

```diff
@@ -0,0 +1,22 @@
+# RUN: not --crash llc -mtriple=aarch64 -o /dev/null -run-pass=none -verify-machineinstrs %s 2>&1 | FileCheck %s --implicit-check-not="Bad machine code"
```

Comment:
Remove g_ from the test name, it's not testing anything related to gMIR verification